tests/testthat/helper.R

```diff
@@ -130,8 +130,7 @@ getPlatformConnectionDetails <- function(dbmsPlatform) {
     # Add drivers
     DatabaseConnector::downloadJdbcDrivers(dbmsPlatform, pathToDriver = jdbcDriverFolder)
     # Table created to avoid collisions
-    randCode <- paste(sample(letters, 4), collapse = "")
-    cohortTable <- paste0("cgc_", gsub("[: -]", "", Sys.Date(), perl = TRUE), "_", randCode)
+    cohortTable <- paste0("ct_", Sys.getpid(), format(Sys.time(), "%s"), sample(1:100, 1))
```

Comment:
@anthonysena I changed this because unit tests were failing on redshift with the previous code. I assume because it wasn't sufficiently unique to generate a unique table name. Using this code seems to have enough different sources of randomness (process id, millisecond time and a random number).